IanGRap/Squish
# New Options Art

Contributors: Austin

Goal:
- To import the new options block art and slightly modify the functionality

What Changed:
- Camera position changed for options menu
- New 3D model and UI layout for options menu
- UIOptionsMenu
    -Changed the functionality such that you have to press the apply button before any settings are changed and you scroll left and right through all of the options, rather than pressing A for some. 
-  GameSettings.cs / ExtensionMethods.cs
    - Added a new particles setting called Party :). It scales up particle emission rate by 3x. The default value is High but I thought this setting could be fun. If y'all don't like it, its easy to undo.

How to Test:
- Load up the lobby scene and make sure that the functionality changes work:
    - You need to press Apply before any settings take effect
    - You need to scroll left and right to select any of the options (instead of pressing A)
